app/src/index.ts

```diff
@@ -114,8 +115,22 @@ function updateTray() {
     { type: 'separator' },
   ]
 
+  const discoveryItems: MenuItemConstructorOptions[] = []
+
+  if (discoveryEnabled) {
+    discoveryItems.push({
+      label: `Network Service: ${process.env.OLLAMA_SERVICE_NAME ?? 'OllamaProvider'}`,
```

Comment:
I wasn't sure if there was a good way to share constant strings between the Golang code in `cmd.go` and the TypeScript code in `index.ts`, so I just wrote `"OllamaProvider"` in both places in case the user doesn't overwrite the default with the `OLLAMA_SERVICE_NAME` environment variable.

We could also probably search for the service ourselves here, but I didn't want to introduce too much complexity for such a small feature that is just displaying some info to the user.